dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -298,10 +286,10 @@ void VanityUtilities::ParseXML(const std::string& file) {
 					continue;
 				}
 
-				VanityObjectLocation locationData;
-				locationData.m_Position = { std::stof(x), std::stof(y), std::stof(z) };
-				locationData.m_Rotation = { std::stof(rw), std::stof(rx), std::stof(ry), std::stof(rz) };
-				locationData.m_Chance = 1.0f;
+				VanityObjectLocation locationData {
+					.m_Position = { std::stof(x), std::stof(y), std::stof(z) },
+					.m_Rotation = { std::stof(rw), std::stof(rx), std::stof(ry), std::stof(rz) },
```

Comment:
try parse here please, though given its a block of these a general try and catch may work better for reporting on a failed to parse value.